### PR TITLE
fix: typos on readme

### PR DESCRIPTION
**Pull request by Gauvino**
Fix typo on README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,8 @@
 <a href="#contributors-"><img alt="All Contributors" src="https://img.shields.io/badge/all_contributors-33-orange.svg"/></a>
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-**Jellyseerr** is a free and open source software application for managing requests for your media library.
-It is a a fork of [Overseerr](https://github.com/sct/overseerr) built to bring support for [Jellyfin](https://github.com/jellyfin/jellyfin) & [Emby](https://github.com/MediaBrowser/Emby) media servers!
+**Jellyseerr** is a free and open-source software application for managing requests for your media library.
```
Comment (fallenbagel) [inline README.md]: Not a typo.

File: README.md [hunk 2]
```diff
@@ -40,11 +40,11 @@ With more features on the way! Check out our [issue tracker](https://github.com/
 
 #### Pre-requisite (Important)
 
-_*On Jellyfin/Emby, ensure the `settings > Home > Automatically group content from the following folders into views such as 'Movies', 'Music' and 'TV'` is turned off*_
+_*On Jellyfin/Emby, ensure the `Settings > Home > Automatically group content from the following folders into views such as 'Movies', 'Music' and 'TV'` is turned off*_
 
 ### Launching Jellyseerr using Docker (Recommended)
 
-Check out our dockerhub for instructions on how to install and run Jellyseerr:
```
Comment (Fallenbagel) [inline README.md:47]: Not a typo.

File: README.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ https://hub.docker.com/r/fallenbagel/jellyseerr
 
 Pre-requisites:
 
-- Nodejs [v18](https://nodejs.org/download/release/v18.18.2)
```
Comment (fallenbagel) [inline README.md]: Not a typo.

File: README.md [hunk 4]
```diff
@@ -85,7 +85,7 @@ _to set env variables such as `JELLYFIN_TYPE=emby` create a file called `.env` i
 cd /opt
 ```
 
-2. Then clone the follow commands to clone and checkout to the stable version
+2. Then execute the followed commands to clone and checkout to the stable version
```
Comment (fallenbagel) [inline README.md]: following*

File: README.md [hunk 5]
```diff
@@ -104,9 +104,9 @@ yarn run build
 5. If you want to run jellyseerr as a _Systemd-service:_
 
 - assuming jellyseerr was cloned to `/opt/`
-- first create the environmentfile at `/etc/jellyseerr/jellyseerr.conf`
+- first create the environment file at `/etc/jellyseerr/jellyseerr.conf`
 
-Environmentfile:
```
Comment (Fallenbagel) [inline README.md:109]: Not a typo.

**Review by Fallenbagel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Gauvino) [reply README.md]: It's an overall, but what do you want separate commit for every fix ?

Comment (fallenbagel) [reply README.md]: Oh wait this was accidentally

File: README.md (hunk 2 above)
Comment (Fallenbagel) [reply README.md:47]: my bad. Its better to have it as docker hub

File: README.md (hunk 3 above)
Comment (fallenbagel) [reply README.md]: @Gauvino this is not a typo. Please do not change the versions.

**Comment by Fallenbagel** [comment]
@Gauvino Could you resolve the suggested changes

File: README.md [hunk 6]
```diff
@@ -53,7 +53,7 @@ https://hub.docker.com/r/fallenbagel/jellyseerr
 
 Pre-requisites:
 
-- Nodejs [v18](https://nodejs.org/download/release/v18.18.2)
+- Nodejs [v18](https://nodejs.org/download/release/v18.19.1)
```
Comment (Gauvino) [inline README.md]:
```suggestion
- Nodejs [v18](https://nodejs.org/download/release/v18.18.2)
```

File: README.md (hunk 3 above)
Comment (Gauvino) [reply README.md]: Removed changed !

**Review by Fallenbagel — APPROVED** [review]

**Pull request merged**